armi/reactor/tests/test_reactors.py

```diff
@@ -54,7 +54,8 @@ def buildOperatorOfEmptyHexBlocks(customSettings=None):
         Dictionary of off-default settings to update
     """
     settings.setMasterCs(None)  # clear
-    cs = settings.getMasterCs()  # fetch new
+    cs = settings.Settings()  # fetch new
+    settings.setMasterCs(cs)  # reset
```

Comment:
Arrielle, please note this is all the `getMasterCs()` call ever did here.  But this gets us one step closer to "removing all getMasterCs, so we can ditch all setMasterCs for free".

Reply:
I'm cool with it :-)